extension/maxminddb.c

```diff
@@ -644,13 +857,30 @@ static PyTypeObject Reader_Type = {
     .tp_dealloc = Reader_dealloc,
     .tp_doc = "Reader object",
     .tp_flags = Py_TPFLAGS_DEFAULT,
+    .tp_iter = Reader_iter,
     .tp_methods = Reader_methods,
     .tp_members = Reader_members,
     .tp_name = "Reader",
     .tp_init = Reader_init,
 };
 // clang-format on
 
+static PyMethodDef ReaderIter_methods[] = {{NULL, NULL, 0, NULL}};
+
+// clang-format off
+static PyTypeObject ReaderIter_Type = {
+    PyVarObject_HEAD_INIT(&PyType_Type, 0)
+    .tp_basicsize = sizeof(ReaderIter_obj),
+    .tp_dealloc = ReaderIter_dealloc,
+    .tp_doc = "Iterator for Reader object",
+    .tp_flags = Py_TPFLAGS_DEFAULT,
+    .tp_iter = PyObject_SelfIter,
+    .tp_iternext = ReaderIter_next,
+    .tp_methods = ReaderIter_methods,
+    .tp_name = "odict_iterator",
```

Comment:
Why this name? The docs suggest to me that it should be something specific to this library/type. I'm not sure.